flow360/component/flow360_params/flow360_params.py

```diff
@@ -73,6 +73,98 @@ class MeshBoundary(Flow360BaseModel):
     no_slip_walls: Union[List[str], List[int]] = pd.Field(alias="noSlipWalls")
 
 
+class TurbulenceValueSpecification(Flow360BaseModel):
+    """Turbulence values on boudaries or freestream
+
+    Parameters
+    ----------
+    turbulentKineticEnergy : non-dimensional [`C_inf^2`]
+        Turbulent kinetic energy. Applicable only when using SST model.
+
+    turbulentIntensity : non-dimensional [`-`]
+        Turbulent intensity. Applicable only when using SST model.
+        This is related to turbulent kinetic energy as: 
+        `turbulentKineticEnergy = 1.5*pow(U_ref * turbulentIntensity, 2)`. 
+        Note the use of the freestream velocity U_ref instead of C_inf.
+
+    specificDissipationRate : non-dimensional [`C_inf/L_gridUnit`]
+        Turbulent specific dissipation rate. Applicable only when using SST model.
+
+    turbulentViscosityRatio : non-dimensional [`-`]
+        The ratio of turbulent eddy viscosity over the freestream viscosity. Applicable for both SA and SST model.
+
+    turbulentLengthScale : non-dimensional [`L_gridUnit`]
+        The turbulent length scale is an estimation of the size of the eddies that are modeled/not resolved. 
+        Applicable only when using SST model. This is related to the turbulent kinetic energy and turbulent
+        specific dissipation rate as: `L_T = sqrt(k)/(pow(beta_0^*, 0.25)*w)` where `L_T` is turbulent length scale,
+        `k` is turbulent kinetic energy, `beta_0^*` is 0.09 and `w` is turbulent specific dissipation rate. 
+        Applicable only when using SST model.
+
+    modifiedTurbulentViscosityRatio : non-dimensional [`-`]
+        The ratio of modified turbulent eddy viscosity (SA) over the freestream viscosity. 
+        Applicable only when using SA model.
+
+    modifiedTurbulentViscosity : non-dimensional [`C_inf*L_gridUnit`]
+        The modified turbulent eddy viscosity (SA). Applicable only when using SA model.
+
+    """
+
+    turbulentKineticEnergy: Optional[NonNegativeFloat] = pd.Field(alias="turbulentKineticEnergy")
+    turbulentIntensity: Optional[NonNegativeFloat] = pd.Field(alias="turbulentIntensity")
+    specificDissipationRate: Optional[NonNegativeFloat] = pd.Field(alias="specificDissipationRate")
+    turbulentViscosityRatio: Optional[NonNegativeFloat] = pd.Field(alias="turbulentViscosityRatio")
+    turbulentLengthScale: Optional[PositiveFloat] = pd.Field(alias="turbulentLengthScale")
+    modifiedTurbulentViscosityRatio: Optional[PositiveFloat] = pd.Field(
+        alias="modifiedTurbulentViscosityRatio"
+    )
+    modifiedTurbulentViscosity: Optional[PositiveFloat] = pd.Field(
+        alias="modifiedTurbulentViscosity"
+    )
```

Comment:
use snake_case for model parameters and camelCase for aliases

Reply:
Done